hazelcast-jet-core/src/main/java/com/hazelcast/jet/Jet.java

```diff
@@ -68,7 +67,9 @@ public static JetInstance newJetInstance(JetConfig config) {
         configureJetService(config);
         HazelcastInstanceImpl hazelcastInstance = ((HazelcastInstanceProxy)
                 Hazelcast.newHazelcastInstance(config.getHazelcastConfig())).getOriginal();
-        return Util.getJetInstance(hazelcastInstance.node.nodeEngine);
+        JetService jetService = hazelcastInstance.node.nodeEngine.getService(JetService.SERVICE_NAME);
+        jetService.getJobCoordinationService().startScanningForJobs();
```

Comment:
can we refactor this to somewhere else, I see the logic duplicated in test code. I think it should be a method on the root service like `onAfterInit` or similar rather than calling coordination service directly.